code/modules/mining/ores_coins.dm

```diff
@@ -531,36 +530,46 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 
 /obj/item/coin/gold
 	custom_materials = list(/datum/material/gold = COIN_MATERIAL_AMOUNT)
+	grind_results = list(/datum/reagent/gold = COIN_MATERIAL_AMOUNT * 0.2)
```

Comment:
Thinking about it from a broad perspective, I don't think using the `_MATERIAL_AMOUNT` defines in grind result is a good idea, as there's a degree of seperation between sheet material quantity (Not that I'm signing up for refactoring these again, yikes) and their chemical reaction counterparts.

I'd prefer you leave these as 5 units outright, OR, make a define for a standardized coin chemical amount, which you can always still split and divide as necessary below.